src/Caching/Abstractions/src/MemoryCacheEntryExtensions.cs

```diff
@@ -24,8 +25,9 @@ public static class MemoryCacheEntryExtensions
         /// <summary>
         /// Sets the size of the cache entry value.
         /// </summary>
-        /// <param name="options"></param>
-        /// <param name="size"></param>
+        /// <param name="options">The option to set the size of.</param>
```

Comment:
Is this ambiguous or misleading? It's not the size of the option. It's the size of the cache entry.